app/models/postcode_local_restriction_search.rb

```diff
@@ -1,16 +1,16 @@
 class PostcodeLocalRestrictionSearch
   UK_POSTCODE_PATTERN = %r{
     \A
-    ([Gg][Ii][Rr] 0[Aa]{2}) # Unusual postcodes
+    (GIR\s?0AA) # Unusual postcodes
     |
     (
       # Outward code, for example SW1A
-      (([A-Za-z][0-9]{1,2})|(([A-Za-z][A-Ha-hJ-Yj-y][0-9]{1,2})|(([A-Za-z][0-9][A-Za-z])|([A-Za-z][A-Ha-hJ-Yj-y][0-9][A-Za-z]?))))
+      (([A-Z][0-9]{1,2})|(([A-Z][A-HJ-Y][0-9]{1,2})|(([A-Z][0-9][A-Z])|([A-Z][A-HJ-Y][0-9][A-Z]?))))
       \s?
-      [0-9][A-Za-z]{2} # Inward code, for example 2AA
+      [0-9][A-Z]{2} # Inward code, for example 2AA
     )
     \Z
-  }x.freeze
+  }xi.freeze
```

Comment:
Is it the `i` that makes the regex case-insensitive?

Reply:
Yup, I've updated the commit message to explain that.